src/Player.hpp

```diff
@@ -0,0 +1,34 @@
+#include <string>
+#include <vector>
+#include <memory>
+#include "Property.hpp"
+class Player{
+
+    std::string color_;
```

Comment:
Maybe it should be enum instead of std::string? String allow us to have any colors.